crml/nft/src/lib.rs

```diff
@@ -689,50 +598,35 @@ decl_module! {
 					Error::<T>::ActivationTimeInvalid
 				);
 				SeriesMassDrops::insert(collection_id, series_id, mass_drop);
-			} else {
```

Comment:
Does this mean people will be able to create an empty series and mint additional in that series?

Reply:
yep, couldn't think of a good reason to prevent this action vs. allow it. since with this you can configure the series then add tokens after